Google.Cloud.EntityFrameworkCore.Spanner/Migrations/Internal/SpannerMigrationCommandExecutor.cs

```diff
@@ -62,9 +63,7 @@ public async Task ExecuteNonQueryAsync(IEnumerable<MigrationCommand> migrationCo
 
         private bool IsDdlStatement(string statement)
         {
-            statement = statement.ToUpper();
-            // https://cloud.google.com/spanner/docs/data-definition-language#ddl_syntax
-            return statement.StartsWith("CREATE") || statement.StartsWith("DROP") || statement.StartsWith("ALTER");
```

Comment:
This check was too simple: If the statement would start with a comment or one or more spaces it would not be correctly recognized as a DDL statement.